src/testRunner/unittests/config/configurationExtension.ts

```diff
@@ -183,7 +183,46 @@ namespace ts {
                     "dev/tests/unit/spec.ts": "",
                     "dev/tests/utils.ts": "",
                     "dev/tests/scenarios/first.json": "",
-                    "dev/tests/baselines/first/output.ts": ""
+                    "dev/tests/baselines/first/output.ts": "",
+                    "dev/configs/extendsArrayFirst.json": JSON.stringify({
+                        compilerOptions: {
+                            allowJs: true,
+                            noImplicitAny: true,
+                            strictNullChecks: true
+                        }
+                    }),
+                    "dev/configs/extendsArraySecond.json": JSON.stringify({
+                        compilerOptions: {
+                            module: "amd"
+                        },
+                        include: ["../supplemental.*"]
+                    }),
+                    "dev/configs/extendsArrayThird.json": JSON.stringify({
+                        compilerOptions: {
+                            module: null, // eslint-disable-line no-null/no-null
+                            noImplicitAny: false
+                        },
+                        include: ["../supplemental.*"]
+                    }),
+                    "dev/configs/extendsArrayFourth.json": JSON.stringify({
+                        compilerOptions: {
+                            module: "system",
+                            strictNullChecks: false
+                        },
+                        include: null, // eslint-disable-line no-null/no-null
+                        files: ["../main.ts"]
+                    }),
+                    "dev/configs/extendsArrayFifth.json": JSON.stringify({
+                        extends: ["./extendsArrayFirst", "./extendsArraySecond", "./extendsArrayThird", "./extendsArrayFourth"],
+                        files: [],
+                    }),
+                    "dev/extendsArrayFails.json": JSON.stringify({
+                        extends: [""],

```

Comment:
Can you instead use a filename that is not present so the error is more comprehensible